YourTurnMyTurn/Services/GroupHelper.cs

```diff
@@ -33,15 +37,25 @@ public Response CreateGroup(string name)
         public Response AddPersonToGroup(string groupId, string personId)
         {
             db.Open();
-            var group = db.Single<Group>(x => x.Id == groupId);
-            if(group.People == null)
-            {
-                group.People = new List<string>();
-            }
-            group.People.Add(personId);
-            db.Update(group);
+            var groupMember = new PersonToGroup { GroupId = groupId, PersonId = personId, Value = 0 };  //TODO: determine how to assign value is group has been running and members have value
+            db.Insert(groupMember);
+            db.Close();
+            return new TextResponse(statusCode: HttpStatusCode.OK, contents: $"{groupMember}");
```

Comment:
What does `groupMember.ToString()` output? I think you just want the ID or something for the return `contents`.